src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/LSPRequestInvoker.cs

```diff
@@ -14,25 +14,36 @@ internal abstract class LSPRequestInvoker
         public abstract Task<TOut> ReinvokeRequestOnServerAsync<TIn, TOut>(
```

Comment:
>ReinvokeRequestOnServerAsync [](http://example.com/codeflow?start=35&length=28)

I thought we weren't going to keep the old signature and we were going to keep the same name, no?